hyperbahn/utils.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
A utils.js file is an anti-pattern; I'm just going to push a commit to inline this function.